modules/logging-set-input-rate-limit.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-flow-control-mechanisms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-set-input-rate-limit_{context}"]
+= Configuring log forwarder input rate limits
+
+You can limit the rate of incoming logs that are collected by configuring the `ClusterLogForwarder` custom resource (CR).
+Input limits can be set on a per-container or per-group basis.
```

Comment:
exclude "per group" for now since it is not in the public API. It exists in the enhancement but was dropped late in development for the "per node" reason I mentioned above